tests/integration/test_concurrency.py

```
@@ -0,0 +1,551 @@
+        # Create service directories
+        for i in range(num_services):
+            service_dir = work_dir / f"vault-service-{i}"
+            service_dir.mkdir()
+            (service_dir / ".env.production").write_text(
+                'DOTENV_PUBLIC_KEY_PRODUCTION="key"\nSECRET="encrypted:..."'
+            )
+
+        env = vault_test_env.copy()
+        env["PYTHONPATH"] = integration_pythonpath
+
+        result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)